.jshintrc

```diff
@@ -23,5 +23,5 @@
 	"sub": true,
 	"trailing": true,
 	"undef": true,
-	"unused": true,
+	"unused": true
```

Comment:
Trailing commas are the norm for our JS code.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
Yes but unfortunately this is not JS, this is JSON, so the trailing comma breaks the parser >_< so we need to remove the comma.